types/config/config.go

```diff
@@ -102,3 +113,13 @@ type PricefeedConfig struct {
 func (p *PricefeedConfig) GetTokens() []types.Token {
 	return p.Tokens
 }
+
+// RewardsFrequencyConfig contains the configuration about rewards frequency in distribution module
+type RewardsFrequencyConfig struct {
```

Comment:
It's probably better to rename this to something like `DistributionConfig` in order to keep it consistent with what it contains (the configuration of the `distribution` module). We might want to use it also for other refresh times in the future as well.